src/styles/card.scss

```diff
@@ -0,0 +1,111 @@
+@import "variables.scss";
+
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```

Comment:
font style: normal is the default value, and font-family better transfer to normalize file.